### PR TITLE
Excluded snekbox/config_pb2.py from coverage

### PR DESCRIPTION
**Pull request by Nightfurex**

### CODE REVIEW

File: .coveragerc [hunk 1]
```diff
@@ -3,6 +3,10 @@ branch = True
 include = snekbox/*
 omit = snekbox/api/app.py
 
+[run]
+branch = True 
+omit = snekbox/config_pb2.py
+
```
Comment (MarkKoz) [inline .coveragerc:9]: You're duplicating the entire `run` section, which I don't think is correct. Instead, you should append the path to the existing `omit` on line 4. To specify multiple omit values, you can put them on separate lines, as shown in the example in the documentation: https://coverage.readthedocs.io/en/6.2/source.html#execution

**Review by MarkKoz — CHANGES_REQUESTED** [review]

**Comment by MarkKoz** [comment]
Thanks for the contribution! Would you mind editing your PR description to mention the issue being fixed? For example, "Fixes #x" where x is the issue number, would associate the PR with that issue. Thus, GitHub would automatically close the issue when the PR is merged.

File: .coveragerc (hunk 1 above)
Comment (Nightfurex) [reply .coveragerc:9]: i am not understanding how to  add multiple omit values do you have any other examples.

Comment (MarkKoz) [reply .coveragerc:9]: Look at how `exclude_lines` is specified currently. Notice how each value is on a new line. The same would apply to `omit` for what you're trying to add.

Comment (Nightfurex) [reply .coveragerc:9]: you mean like these omit = 
                                              snekbox/api/app.py
                                              snekbox/config_pb2.py

Comment (MarkKoz) [reply .coveragerc:9]: Yes, though please indent the lines with 4 spaces, just like the values for `exclude_lines` are indented.

File: .coveragerc [hunk 2]
```diff
@@ -1,7 +1,11 @@
 [run]
 branch = True
 include = snekbox/*
-omit = snekbox/api/app.py
+omit = 
+    snekbox/api/app.py
+    snekbox/config_pb2.py
+
+
```
Comment (MarkKoz) [inline .coveragerc:8]: Remove these 2 extra newlines please.

**Review by MarkKoz — CHANGES_REQUESTED** [review]

**Comment by Nightfurex** [comment]
> Thanks for the contribution! Would you mind editing your PR description to mention the issue being fixed? For example, "Fixes #x" where x is the issue number, would associate the PR with that issue. Thus, GitHub would automatically close the issue when the PR is merged.

so

**Pull request closed without merging**